README.md

```diff
@@ -31,7 +32,9 @@ You can either clone the git repo with:
 ```
 git clone https://github.com/pgmpy/pgmpy
 ```
-or download a zip from: https://github.com/pgmpy/pgmpy/archive/dev.zip
+or download a zip from: [https://github.com/pgmpy/pgmpy/archive/dev.zip](https://github.com/pgmpy/pgmpy/archive/dev.zip)
+
+Alternatively, one can download a .tar.gz file from the PyPI archive [https://pypi.python.org/pypi/pgmpy](https://pypi.python.org/pypi/pgmpy)
```

Comment:
@souravsingh Mention that it would be the last release version and not the latest code.